### PR TITLE
Add create list form

### PR DESCRIPTION
**Pull request by dangalan**

### CODE REVIEW

File: api/airtable.ts [hunk 1]
```diff
@@ -9,38 +9,20 @@ export class AirtableClient {
...
-    records.forEach(record => {
-      const object: {[key: string]: any} = {}
-      keys.forEach(key => {
-        object[key.toString()] = record.get(key.toString())
-      })
-      objects.push(object as T)
-    })
-
-    return objects as T[]
+    return records.map(record => record.fields) as T[]
```
Comment (dangalan) [inline api/airtable.ts:26]: There was no need to iterate through a list of keys to get the fields we needed from each Airtable record 🤦🏼

Comment (btmoore4) [reply api/airtable.ts:26]: lol

File: hooks/useCreateList.ts [hunk 2]
```diff
@@ -1,13 +1,42 @@
...
+  creator: string
+  description: string
+}
+
+const fieldRequiredMessage = 'Cannot be blank'
+
+const createListSchema = yup.object({
+  name: yup.string().required(fieldRequiredMessage),
+  creator: yup.string().required(fieldRequiredMessage),
+  description: yup.string().required(fieldRequiredMessage),
```
Comment (btmoore4) [inline hooks/useCreateList.ts:21]: Just want to make sure, is this where the list is created? Don't see where we add the services ids

**Review by btmoore4 — APPROVED** [review]

File: hooks/useCreateList.ts (hunk 2 above)
Comment (dangalan) [reply hooks/useCreateList.ts:21]: So this schema is just for the form fields, we'll grab service ids from the `selectedFields` state below. Once we merge your PR in I can integrate the create API call with the form.

**Pull request merged**